### DISCUSSION TITLE
python: can't open file '/app/manage.py': [Errno 2] No such file or directory

### DISCUSSION THREAD

**Discussion by phoenixhackt** [cookiecutter/cookiecutter-django#4779] [Q&A]
Hello, I'm writing to you because I'm experiencing a problem that I've been trying to solve for several days but which I'm really unable to overcome using cookiecutter. I'm doing all the steps as explained in the documentation and in the few videos I've seen on the internet. I set the parameters with Docker and Postrges15 but when I run the command: 
```shell 
docker compose -f local.yml up -d 
``` 
Then the django and docs container doesn't launch and I get the following in the logs: 
```shell 
Waiting for PostgreSQL to become available... 2024-01-08T18:56:08.078170950Z PostgreSQL is available 2024-01-08T18:56:08.115577061Z python: can't open file '/app/manage.py': [Errno 2] No such file or directory 
````

**Comment by luzfcb** [comment]
Questions:
1. What operating system are you using?
2. Is there a `manage.py` file in the root directory of your project?
3. Did you build the docker image first? What happens if you run` docker compose up` after` docker compose build`?

```
docker compose -f local.yml build --no-cache
docker compose -f local.yml up -d 
```

If the command `docker compose -f local.yml build --no-cache` gets an error, please send the logs and your local.yml file

> Note: remember to anonymize the project name (project slug) and other private information before send any logs/files

**Reply by phoenixhackt** [reply]
1. Windows 11
2. Yes
3. I launched this command 
```shell
docker compose -f local.yml build
```  
before this one 
```shell
docker compose -f local.yml up -d 
```  
but without the --no-cache. 
So I try and come back to you.
Thanks for the quick response.

**Reply by phoenixhackt** [reply]
Ok I still get the same logs for Django and docs, postgres still works:
```shell
Waiting for PostgreSQL to become available...
2024-01-08T20:06:37.839758449Z PostgreSQL is available
2024-01-08T20:06:37.908607149Z python: can't open file '/app/manage.py': [Errno 2] No such file or directory
``` 
```shell
make: *** No rule to make target 'livehtml'.  Stop.
```

**Reply by luzfcb** [reply]
Interesting.  Could you tell me which settings you chose to start the project?  Are you using docker engine installed inside wsl2, or are you using Docker Desktop?  I would probably set up a Windows VM to try and reproduce the problem.

**Reply by phoenixhackt** [reply]
I use Docker Desktop.
And I use this settings :
```
project_name [My Awesome Project]: Project Name
project_slug [reddit_clone]: projectname
description [Behold My Awesome Project!]: Project description.
author_name [Daniel Roy Greenfeld]: FirstName LastName
domain_name [example.com]: project-name.com
email [daniel-greenfeld@example.com]: project-name@outlook.com
version [0.1.0]: 0.0.1
Select open_source_license:
1 - MIT
2 - BSD
3 - GPLv3
4 - Apache Software License 2.0
5 - Not open source
Choose from 1, 2, 3, 4, 5 [1]: 5
Select username_type:
1 - username
2 - email
Choose from 1, 2 [1]: 2
timezone [UTC]: Paris/France
windows [n]: n
Select an editor to use. The choices are:
1 - None
2 - PyCharm
3 - VS Code
Choose from 1, 2, 3 [1]: 2
use_docker [n]: y
Select postgresql_version:
1 - 15
2 - 14
3 - 13
4 - 12
5 - 11
6 - 10
Choose from 1, 2, 3, 4, 5 [1]: 1
Select cloud_provider:
1 - AWS
2 - GCP
3 - None
Choose from 1, 2, 3 [1]: 1
Select mail_service:
1 - Mailgun
2 - Amazon SES
3 - Mailjet
4 - Mandrill
5 - Postmark
6 - Sendgrid
7 - SendinBlue
8 - SparkPost
9 - Other SMTP
Choose from 1, 2, 3, 4, 5, 6, 7, 8, 9 [1]: 3
use_async [n]: n
use_drf [n]: y
Select frontend_pipeline:
1 - None
2 - Django Compressor
3 - Gulp
4 - Webpack
Choose from 1, 2, 3, 4 [1]: 1
use_celery [n]: n
use_mailpit [n]: n
use_sentry [n]: n
use_whitenoise [n]: y
use_heroku [n]: n
Select ci_tool:
1 - None
2 - Travis
3 - Gitlab
4 - Github
Choose from 1, 2, 3, 4 [1]: 4
keep_local_envs_in_vcs [y]: y
debug [n]: n
```

**Reply by phoenixhackt** [reply]
Hello, for those who have the same problem, it seems to be due to the fact that I was running cookiecutter on a USB key, because I did it on a local folder and everything works perfectly.
